tests/integration/security/testdata/beta-mtls-automtls-workload.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: security.istio.io/v1beta1
+kind: PeerAuthentication
+metadata:
+  name: "a-disable"
+  annotations:
+    test-suite: "beta-mtls-automtls-workload"
+spec:
+  selector:
+    matchLabels:
+      app: a
+  mtls:
+    mode: DISABLE
+---
+apiVersion: security.istio.io/v1beta1
+kind: PeerAuthentication
+metadata:
+  name: "b-strict"
+  annotations:
+    test-suite: "beta-mtls-automtls-workload"
+spec:
+  selector:
+    matchLabels:
+      app: b
+  mtls:
+    mode: STRICT
+---
+apiVersion: security.istio.io/v1beta1
+kind: PeerAuthentication
+metadata:
+  name: "c-per-port"
+  annotations:
+    test-suite: "beta-mtls-automtls-workload"
+spec:
+  selector:
+    matchLabels:
+      app: c
+  mtls:
+    mode: STRICT
+  portLevelMtls:
+    # 8090 is the targetPort for service http(80)
```

Comment:
8090 or 8080?

Reply:
8090 is the http port, would just works fine, right?
https://github.com/istio/istio/blob/08f3ac1e1b7b8204dcc8fba2c0e0da5f6039536a/tests/integration/security/util/framework.go#L82

Reply:
oh I thought this is commenting for the port number 8080 in line 41, maybe update the comment to make it clear

Reply:
good point. updated to make the same.